docs/src/main/asciidoc/_admin-guide/management-tasks/YAML_Configuration_file.adoc

```diff
@@ -1,19 +1,17 @@
 [[YAML_Configuration_file]]
 = (YAML) Configuration file (Experimental))
 
-In order to provide some way to separate customization from the configuration provisionned by Galleon we want to *experiment* with a new feature where this customization is provided by YAML files.
-Because this feature is *EXPERIMENTAL* we are disabling it by default and you can *NOT* rely on it being present or compatible in the future releases.
+Provides a way to separate customization from the configuration provisioned by Galleon by using YAML configuration files.
+Because this feature is *EXPERIMENTAL* it may removed in future releases.
```

Comment:
This feature is not experimental any more.

Reply:
Removed `experimental`